src/Google.Cloud.Metadata.V1/AccessConfigurationType.cs

```diff
@@ -0,0 +1,33 @@
+﻿// Copyright 2016 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+namespace Google.Cloud.Metadata.V1
+{
+    /// <summary>
+    /// The type of access configuration for a network interface.
+    /// </summary>
+    /// <seealso cref="AccessConfigurationMetadata.NetworkType"/>
+    public enum AccessConfigurationType
+    {
+        /// <summary>
+        /// The network interface uses a one-to-one NAT access configuration.
+        /// </summary>
+        OneToOneNat,
+
+        /// <summary>
+        /// Unknown access configuration type.
+        /// </summary>
+        Unknown = -1
```

Comment:
@jskeet had mentioned in a comment above that it might be good to make all Unknowns -1, but 0 might be better so anything which isn't initialized gets the value unknown by default. Jon, are you ok with this?